### PR TITLE
Dev 492 Encrypt subjects

### PR DESCRIPTION
**Pull request by atifsaddique211f**

### CODE REVIEW

File: src/assets/static/pgp-worker.js [hunk 1]
```diff
@@ -54,25 +56,21 @@ onmessage = function (event) {
...
-                                    decrypted: true,
-                                    callerId: event.data.callerId
-                                });
-                            });
-                    } else {
-                        postMessage({decryptedContent: data, decrypted: true, callerId: event.data.callerId});
-                    }
+            decryptContent(event.data.mailData.content, decryptedPrivKeys[event.data.mailboxId]).then((content) => {
+                decryptContent(event.data.mailData.subject, decryptedPrivKeys[event.data.mailboxId]).then((subject) => {
+                    decryptContent(event.data.mailData.incomingHeaders, decryptedPrivKeys[event.data.mailboxId]).then((incomingHeaders) => {
```
Comment (Waseem826) [inline src/assets/static/pgp-worker.js]: Are `incomingHeaders` always present in mail data now? The `if` condition has been removed.

Comment (atifsaddique211f) [reply src/assets/static/pgp-worker.js]: it has been handled inside decrypt content function rather than applying this condition for everyone separately.

Comment (atifsaddique211f) [reply src/assets/static/pgp-worker.js]: `function decryptContent(data, privKeyObj) {
    if (!data) {
        return Promise.resolve(data);
    }`

Comment (Waseem826) [reply src/assets/static/pgp-worker.js]: ah, okay

**Pull request merged**